library/Notifications/Widget/Timeline/Entry.php

```diff
@@ -19,8 +17,8 @@ class Entry extends TimeGrid\Entry
     /** @var Member */
     protected $member;
 
-    /** @var ?HtmlDocument Content of the flyoutmenu that is shown when the entry is hovered */
-    protected ?HtmlDocument $flyoutContent;
+    /** @var ?EntryFlyout Content of the flyoutmenu that is shown when the entry is hovered */
+    protected ?EntryFlyout $flyoutContent;
```

Comment:
If it's nullable type, it must be initialized explicitly with null, as of PHP 8.4